stonesoup/models/measurement/nonlinear.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, *args, **kwargs):
         for model in self.model_list:
             if model.ndim_state != self.ndim_state:
                 raise ValueError("Models must all have the same `ndim_state`")
+            if model.covar(**kwargs) is None:
```

Comment:
Not sure this should be called here, certainly not with `kwargs`. I think check in `rvs` and `pdf` is fine.